src/addons/transitions/ReactTransitionChildMapping.js

```diff
@@ -25,6 +25,9 @@ var ReactTransitionChildMapping = {
     if (!children) {
       return children;
     }
+    // TODO: `flattenChildren` now takes an extra `selfDebugID` argument
```

Comment:
```
var internalInstance = ReactInstanceMap.get(this);
internalInstance._debugID
```

should work. Maybe someday we'll have a public API to get this that anyone could use but for now I think it's okay for us to get the internal instance for this here.